eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version1/ChannelCodecs1.scala

```diff
@@ -79,6 +79,15 @@ private[channel] object ChannelCodecs1 {
 
     def setCodec[T](codec: Codec[T]): Codec[Set[T]] = listOfN(uint16, codec).xmap(_.toSet, _.toList)
 
+    val updateAddHtlcCodec: Codec[UpdateAddHtlc] = (
+      ("channelId" | bytes32) ::
+        ("id" | uint64overflow) ::
+        ("amountMsat" | millisatoshi) ::
+        ("paymentHash" | bytes32) ::
+        ("expiry" | cltvExpiry) ::
+        ("onionRoutingPacket" | OnionCodecs.paymentOnionPacketCodec) ::
+        ("tlvStream" | provide(TlvStream.empty[UpdateAddHtlcTlv]))).as[UpdateAddHtlc]
+
```

Comment:
Why did you have to define this one here, but were able to use the regular `lightningMessageCodec` for the `updateMessageCodec` a few lines below?

Reply:
This was indeed unnecessary, good catch! Removed in https://github.com/ACINQ/eclair/pull/1891/commits/003c0db9cc9c6c1263fa5639bdb58c61e7e7641e